docs/src/Groups/grouphom.md

```diff
@@ -167,8 +167,7 @@ preimage(f::GAPGroupHomomorphism{S, T}, H::T) where S <: GAPGroup where T <: GAP
 ## Groups created by isomorphisms
```

Comment:
I really don't like this section title. Though I also don't have a great alternative suggestion... perhaps just
```suggestion
## Group isomorphisms
```